nicsdru_nidirect_theme.theme

```diff
@@ -66,13 +66,23 @@ function nicsdru_nidirect_theme_preprocess_field(&$variables) {
     $content_type = $variables['element']['#bundle'];
     switch ($content_type) {
       case 'health_condition':
-        _build_audit_link('health_condition', $variables);
+        $msg = _build_audit_link('health_condition', $variables);
+        if ($msg) {
+          // Send out 'needs audit' link as a Drupal warning message.
+          // N.B. This will only be visible if the 'System Messages' block is visible.
+          \Drupal::messenger()->addWarning(t($msg->jsonSerialize()));
```

Comment:
This just seems odd to me, passing json into t()?

Reply:
I agree, but it seems that it has to be done. If I just pass the HTML of the Link to the Drupal messenger then it is displayed to the user as raw HTML. The only way that I could find to avoid this is to run it through t() on the way in. Any suggestions welcome .........

Reply:
ahh pretty weird but if it works, it works 👍